crates/holochain/src/core/queue_consumer/sys_validation_consumer.rs

```diff
@@ -32,23 +32,17 @@ pub fn spawn_sys_validation_consumer(
         conductor.task_manager(),
         (tx.clone(), rx),
         move || {
-            if let Some(representative_agent) =
-                get_representative_agent(&conductor, &network.dna_hash())
-            {
-                Either::Left(sys_validation_workflow(
-                    workspace.clone(),
-                    current_validation_dependencies.clone(),
-                    trigger_app_validation.clone(),
-                    trigger_publish.clone(),
-                    trigger_self.clone(),
-                    network.clone(),
-                    keystore.clone(),
-                    representative_agent,
-                ))
-            } else {
-                tracing::warn!("No agent found for DNA, skipping sys validation");
-                Either::Right(async move { Ok(WorkComplete::Complete) })
-            }
+            let representative_agent = get_representative_agent(&conductor, &network.dna_hash());
+            sys_validation_workflow(
+                workspace.clone(),
+                current_validation_dependencies.clone(),
+                trigger_app_validation.clone(),
+                trigger_publish.clone(),
+                trigger_self.clone(),
+                network.clone(),
+                keystore.clone(),
+                representative_agent,
```

Comment:
Convert this to an `Option<AgentPubKey>`, so that sys validation always runs. Issuing a warrant will fail if the representative agent is `None`, but that shouldn't prevent sys validation from running.